### ISSUE TITLE
What is the future of this project?

### ISSUE THREAD

**Issue by metalrex100** [samber/lo#78]
Is this project going to be actively maintained?

I see that people would like to contribute, but issues and PR's review and acceptance depends only on @samber and his free time.
May be such bus-factor can be reduced somehow?

**Comment by affanshahid** [comment]
I agree there are a bunch of PRs and issues with no responses. I think this is a great project and @samber should on-board more contributors to help maintain.

**Comment by mvrahden** [comment]
Or move the project into its own organization?

**Comment by gopherhw** [comment]
Any other progress?
@samber
